feature-utils/poly-look/test/react-components/chip.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import Chip from "../../src/react-components/chip.jsx";
+
+const chipId = "chipId";
+const mockedHandleClick = jest.fn();
+
+const chip = <Chip id={chipId} handleClick={mockedHandleClick} active />;
+
+it("Chip can be clicked", () => {
```

Comment:
I think you're supposed to wrap these `it` calls in a `describe` call, something like:

```
describe("Chip", () => {
    it("Can be clicked"), () => {
        ...
    });
    ...
});
```

But if it works without that in Jest, I don't mind if we keep it like this at first.

Reply:
You can create groups of tests with that, and also add pre and post-conditions. In absence of those, these tests work without a problem. We do have both styles in the repo.

Reply:
Exactly, as @JJ has said, these tests work well in Jest without being wrapped in a `describe`. In addition, when running the tests, it already indicates to which component the tests belong to - so adding a description repeating the name of the component might be redundant 🤔 . 

I'll leave it as it is right now, but if you @fhd think it will be clearer and easier to understand if I add the description, I don't have any problem in adding it.

Reply:
Your call, really. It does not make a difference in this case.